### DISCUSSION TITLE
Browse files on remote via sftp

### DISCUSSION THREAD

**Discussion by tony-sol** [gokcehan/lf#882] [Q&A] [answered]
Is there way to connect to remote machine with lf as easy as it is with midnight-commander?

**Comment by DusanLesan** [comment, marked as answer]
Not sure how easy it is with midnight-commander but I just mount remote using sshfs to some local directory. I do not do mounting from lf but that should not be too hard.

**Reply by DusanLesan** [reply]
You could write something like this but with some validations to mount from lf:
```
cmd mountssh ${{
	echo "Host:"
	read host
	echo "Target:"
	read target
	mkdir "mnt-$host"
	sshfs $host:$target "mnt-$host"
}}

map ms mountssh
```

**Reply by tony-sol** [reply]
With mc i can explore whole available file system on remote, but with sshfs, is it only mounted directory accessible?

**Reply by DusanLesan** [reply]
The target should control that. If you leave it empty, it will mount at home but if you set it to `/` it should mount root
